temporalcli/commandsmd/commands.md

```diff
@@ -378,6 +379,113 @@ However, URI values for archival states cannot be changed after the states are e
+
+This will fail if an endpoint with the same name is already registered.
+
+```
+temporal operator nexus endpoint create \
+  --name my-endpoint \
+  --target-namespace my-namespace \
+  --target-task-queue my-task-queue \
+  --description-file DESCRIPTION.md
+```
+
+#### Options
+
+* `--name` (string) - Endpoint name. Required.
+* `--description` (string) - Endpoint description in markdown format (encoded using the configured codec server).
+* `--description-file` (string) - Endpoint description file in markdown format (encoded using the configured codec server).
+* `--target-namespace` (string) - Namespace in which a handler worker will be polling for Nexus tasks on.
+* `--target-task-queue` (string) - Task Queue in which a handler worker will be polling for Nexus tasks on.
+* `--target-url` (string) - URL to direct Nexus requests to.
+
+### temporal operator nexus endpoint delete: Delete a Nexus Endpoint (EXPERIMENTAL).
+
+The temporal operator nexus endpoint delete deletes a Nexus Endpoint configuration from the Server.
+
+```
+temporal operator nexus endpoint delete --name my-endpoint
+```
+
+#### Options
+
+* `--name` (string) - Endpoint name. Required.
+
+### temporal operator nexus endpoint get: Get a Nexus Endpoint by name (EXPERIMENTAL).
+
+The temporal operator nexus endpoint get gets a Nexus Endpoint configuration by name from the Server.
+
+```
+temporal operator nexus endpoint get --name my-endpoint
+```
+
+#### Options
+
+* `--name` (string) - Endpoint name. Required.
+
+### temporal operator nexus endpoint list: List Nexus Endpoints (EXPERIMENTAL).
+
+The temporal operator nexus endpoint list lists all Nexus Endpoint configurations on the Server.
+
+```
+temporal operator nexus endpoint list
+```
+
+### temporal operator nexus endpoint update: Update an existing Nexus Endpoint (EXPERIMENTAL).
+
+The temporal operator nexus endpoint update command updates an existing Nexus Endpoint on the Server.
+
+An endpoint name is used by in workflow code to invoke Nexus operations.
+The endpoint target may either be a worker, in which case `--target-namespace` and `--target-task-queue` must both be
+provided, or an external URL, in which case `--target-url` must be provided.
+
+The endpoint is patched leaving any existing fields for which flags are not provided as they were.
+
+**Update only the target task queue**:
+
+```
+temporal operator nexus endpoint update \
+  --name my-endpoint \
+  --target-task-queue my-other-queue
+```
+
+**Update only the description**:
+
+```
+temporal operator nexus endpoint update \
+  --name my-endpoint \
+  --description-file DESCRIPTION.md
+```
+
+#### Options
+
+* `--name` (string) - Endpoint name. Required.
+* `--description` (string) - Endpoint description in markdown format (encoded using the configured codec server).
+* `--description-file` (string) - Endpoint description file in markdown format (encoded using the configured codec server).
```

Comment:
This was removed in cloud API at https://github.com/temporalio/api-cloud/pull/23#discussion_r1595946337, I think we should remove in this one too. Or we can have description, but I don't think we should get specific with the format expected at this time (I suspect there is a subset of markdown we will start using as part of https://github.com/temporalio/features/issues/486).

Reply:
We may need to discuss this more. There's a strict requirement to have this description. We have it in all of the docs.
I've replied in that cloud API thread to request bringing the description back.

Reply:
Is there a strict requirement to have the description, or a strict requirement to say descriptions are markdown formatted. My issue is more with the latter and I think it should at least be subject to discussion.

Reply:
Yeah, we want markdown. Maybe it's worth consolidating on summary and details to match workflow metadata, I would be okay changing that now.

Reply:
I am not sure we can generally say we want all long-form "description" text accepted by Temporal APIs to be full markdown, and whatever we do want Temporal long-form format to be should be centrally/generally documented. Setting a long-form format specific to Nexus descriptions and not for Temporal as a whole I don't think is the best approach.